test/smoke/win32.test.js

```diff
@@ -38,4 +41,13 @@ _describe('win32', () => {
       assert.match(p.stdout, /Windows 'rulez!'/)
     })
   })
+
```

Comment:
let's _keep_ cov treshold. Add plz smth like core.test.js:

```ts
test('usePwsh() sets proper defaults', () => {
   const wsync = which.sync
   which.sync = bin => bin === 'pwsh' ? 'pwsh' : wsync(bin)
   
   usePwsh()
   assert.equal($.shell, 'pwsh')
   assert.equal($.prefix, '...')
   // ...
   which.sync = wsync
})
```

Reply:
Instead of this win32 smoke test, right?

Reply:
as addition